sending-email.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Sending email
+layout: recommendation
+expires: 2017-11-01
+---
+
+
+## User needs for sending email
+
+- applications notifying users about progress or actions to take
+- automated notifications sent to engineers by systems such as cron or jenkins remembering that we [don’t rely on emails as an automated alerting method for incident responders](https://alphagov.github.io/gds-tech/alerting.html#principles-for-setting-up-an-alerting-system)
```

Comment:
I don't get this, and don't think I agree with it. Results of cron jobs should be written, aggregated, and alerted through logs and associated systems. CI notifications shouldn't be forced via SES unless it's appropriate.